source/working_remotely.rst

```diff
@@ -0,0 +1,60 @@
+################
+Working Remotely
+################
+
+Some nao robots are always turned on in the Robotics Lab for working remotely via the Kodos server.
+
+*****************************
+Connecting to Kodos using SSH
+*****************************
+
+First, your computer should be connected to Kodos using SSH.
+
+.. note::
+    For creating your account on Kodos, please :ref:`contact` us.
+
+Run the following command in a terminal of your computer to connect to Kodos.
+
+.. code-block:: bash
+
+   $ ssh -L 5903:localhost:5903 username@kodos.cse.unsw.edu.au
```

Comment:
you should mention where the `5903` comes from

Reply:
New changes were made to this file.